commitlint.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     },
     parserPreset: {
         parserOpts: {
-            headerPattern: /^(\w*)\((\w.*)\): (DSW-\d{1,7}) (\w.*)$/,
+            headerPattern: /^(\w*)\((\w.*)\): (DSW-(?!0+)\d{1,7}) (\w.*)$/,
```

Comment:
Unrelated to this change, but wonder if we should utilise `\w` rather than `DSW` for tickets that don't below to us such as external contributions

Reply:
Implemented this change after discussion with the team